### DISCUSSION TITLE
Hardcore using a .yaml file.

### DISCUSSION THREAD

**Discussion by TheAdroitDBA** [itzg/docker-minecraft-server#1664] [Q&A] [answered]
Hopefully I'm just missing something simple.   I have been trying to get hardcore mode to work with this yaml config.   It keeps defaulting to survival mode.

`version: "3"

services:
  minecraft:
    image: itzg/minecraft-server
    volumes:
      - minecraftdata:/data
      - /home/minecraft/mods:/mods
      - /home/minecraft/plugins:/plugins
    ports:
      - 25565:25565
    environment:
      VERSION: "1.18.2"
      EULA: "TRUE"
      HARDCORE: "true"
      TYPE: "FABRIC"
      ENABLE_RCON: "true"
      RCON_PASSWORD: "testing"
      RCON_PORT: 28017
      MEMORY: 4G
    tty: true
    stdin_open: true
    restart: unless-stopped
volumes:
  minecraftdata:
    external: true`

**Comment by itzg** [comment, marked as answer]
Add `OVERRIDE_SERVER_PROPERTIES: "true"` to make sure the server properties is reflecting the container config as you change it. Hopefully that's all it is.

**Reply by TheAdroitDBA** [reply]
That did the trick, thank you!
